pkg/template/template.go

```diff
@@ -207,43 +206,16 @@ func timeAgo(ago time.Duration) string {
 		return "just now"
```

Comment:
Kind of weird that we don't use `FuzzyAgo` here. Perhaps we should be? The current functionality was brought over from `gh`.